specs/agents/tracing-instrumentation-db.md

```diff
@@ -1,34 +1,178 @@
+|`context.db.rows_affected`| The number of rows / entities affected by the corresponding db statement / query.| :x: |
+| <hr/> |<hr/>|<hr/>|
+|`context.destination.address`|The hostname / address of the database.| :x: |
+|`context.destination.port`|The port under which the database is accessible.| :x: |
+|`context.destination.service.name`| The `destination.service.name` is used to denote "sameness" of the service. E.g. multiple instances of Oracle databases have all the same name `oracle`. For databases and storages the same value as for the `span.subtype` should be used.| :white_check_mark:|
+|`context.destination.service.type`| Should be the same as the `span.type`. Value: `db`| :white_check_mark:|
+|`context.destination.service.resource`|  Used to detect unique destinations from each service. This field should contain all information that is needed to differentiate different database / storage instances (e.g. in the service map). See details below on how to set this field for specific technologies.| :white_check_mark:|
+|`context.destination.cloud.region`| The cloud region in case the datastore is hosted in a public cloud or is a managed datasatore / database. E.g. AWS regions, such as `us-east-1` | :x: |
 
-For Elasticsearch search-type queries, the request body may be recorded. Alternatively, if a query is specified in HTTP query parameters, that may be used instead. If the body is gzip-encoded, the body should be decoded first.
 
-### Database span names
+## Specific Databases
 
-For SQL operations we perform a limited parsing the statement, and extract the operation name and outer-most table involved (if any). See more details here: https://docs.google.com/document/d/1sblkAP1NHqk4MtloUta7tXjDuI_l64sT2ZQ_UFHuytA.
+### AWS DynamoDb
 
-For Redis, the the span name can simply be set to the command name, e.g. `GET` or `LRANGE`.
+| Field | Value / Examples | Comments |
+|-------|:---------------:|----------|
+|`name`| e.g. `DynamoDB UpdateItem my_table`|  The span name should capture the operation name (as used by AWS for the action name) and the table name, if available. The format should be `DynamoDB <ActionName> <TableName>`. |
+|`type`|`db`|
+|`subtype`|`dynamodb`|
+|`action`| `query` | 
+| __**context.db._**__ |<hr/>|<hr/>|
+|`_.instance`| e.g. `us-east-1` | The AWS region where the table is. |
+|`_.statement`| e.g. `ForumName = :name and Subject = :sub` | For a DynamoDB Query operation, capture the KeyConditionExpression in this field. |
+|`_.type`|`dynamodb`|
+|`_.user`| :heavy_minus_sign: |
+|`_.link`| :heavy_minus_sign: |
+|`_.rows_affected`| :heavy_minus_sign: |
+| __**context.destination._**__ |<hr/>|<hr/>|
+|`_.address`|e.g. `dynamodb.us-west-2.amazonaws.com`|
+|`_.port`|e.g. `5432`|
+|`_.service.name`| `dynamodb` |
+|`_.service.type`|`db`|
+|`_.service.resource`| `dynamodb` |
+|`_.cloud.region`| e.g. `us-east-1` | The AWS region where the table is, if available. |
+  
+### AWS S3
```

Comment:
[minor] This section duplicates information from `tracing-instrumentation-aws.md`. A canonical location would be a little nicer.